README.md

```diff
@@ -9,21 +9,59 @@ server is deployed at:
 
 https://go-acd.appspot.com
 
-# Install
+# Usage
+
+In order to use this library, you must authenticate through the token server. A  (e.g. https://go-acd.appspot.com).
 
-This project is go-gettable:
+Get and build:
 
 ```
-go get gopkg.in/acd.v0/...
+$ go get gopkg.in/acd.v0/cmd/acd
+$ go build gopkg.in/acd.v0/cmd/acd
```

Comment:
@dsoprea that's how `go get` works, it fetches the library and then the dependencies and then installs it.

```
# go get -h
usage: get [-d] [-f] [-fix] [-insecure] [-t] [-u] [build flags] [packages]

Get downloads the packages named by the import paths, along with their
dependencies. It then installs the named packages, like 'go install'.
```